README.md

```diff
@@ -2,6 +2,70 @@
 
 A project that presents an iRODS Zone as HTTP v1.1.
 
+## Quickstart (Running from Docker Hub Image)
+
+1. Generate a local configuration JSON file.
+
+```
+docker run irods/irods_http_api --dump-config-template > config.json
+```
+
+2. Edit/update the template `config.json` file (point to your iRODS Zone, remove OIDC and TLS sections, etc.).
+
+```
+vim config.json
+```
+
+3. Launch the HTTP API with your customized configuration file to check for success/errors:
+
+```
+docker run --rm --name irods_http_api -v ./config.json:/config.json:ro -p 9000:9000 irods/irods_http_api
+```
+
+4. Launch in daemonized form:
+
+```
+docker run -d --rm --name irods_http_api -v ./config.json:/config.json:ro -p 9000:9000 irods/irods_http_api
```

Comment:
Consider wrapping the lines using a trailing backslash.

Reply:
will do.